### PR TITLE
Sorts meetings and adds links to them

### LINKED ISSUE #635: Improve the account dialog
- [x] Sort the meetings of a user in lexicographically order
- [x] Make a link to every user's meeting
- [x] Edit the own profile: On the orga-layer, only the fields `email` and `username` are allowed to change. Switching to a meeting, suddenly all fields are allowed to change... (Fixed by #644)

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Fixes #635

### CODE REVIEW

**Comment by tsiegleauq** [comment]
> Hint: Currently, we have no routine to re-subscribe to the AU-service. Therefore, if a user stays on the start-page in any meeting and navigates to another meeting (by their account-dialog), they will see an empty page.

I suppose thats partially because if you dont reload a component, neither the constructor not the init will be called again. There would be a routine to renew the subscription, using the `this.localModelSubscriptions[subscriptionName]` from `base-model-context-component`

**Review by tsiegleauq — CHANGES_REQUESTED** [review]
>  start-page in any meeting and navigates to another meeting (by their account-dialog), they will see an empty page.

I suppose this can be (and should) be done using routers. navigationEnd (https://angular.io/api/router/NavigationEnd) and some adjustments to the component.

**Review by emanuelschuetze — APPROVED** [review]
Ok for me. Please check the hint of @tsiegleauq

File: client/src/app/site/common/components/start/start.component.ts [hunk 1]
```diff
@@ -57,15 +63,8 @@ export class StartComponent extends BaseModelContextComponent implements OnInit
...
-                this.startContent.welcome_title = welcomeTitle;
-            }),
-            this.meetingSettingsService.get(`welcome_text`).subscribe(welcomeText => {
-                this.startContent.welcome_text = this.translate.instant(welcomeText);
-            }),
-            this.activeMeetingService.meetingObservable.subscribe(meeting => {
-                this.meeting = meeting;
-            })
+            this.router.events.pipe(filter(event => event instanceof NavigationEnd)).subscribe(() => this.refresh()),
+            this.activeMeetingService.meetingObservable.subscribe(meeting => (this.meeting = meeting))
```
Comment (tsiegleauq) [inline client/src/app/site/common/components/start/start.component.ts:67]: they are probably unconcurrent, so you can use `combineLatest` to limit the triggers

File: client/src/app/site/common/components/start/start.component.ts [hunk 2]
```diff
@@ -76,7 +75,23 @@ export class StartComponent extends BaseModelContextComponent implements OnInit
...
+            this.meetingSettingsService.get(`welcome_title`).subscribe(welcomeTitle => {
+                this.startContent.welcome_title = welcomeTitle;
+            })
+        );
+        this.updateSubscription(
+            WELCOME_TEXT_SUBSCRIPTION_NAME,
+            this.meetingSettingsService.get(`welcome_text`).subscribe(welcomeText => {
+                this.startContent.welcome_text = this.translate.instant(welcomeText);
+            })
+        );
```
Comment (tsiegleauq) [inline client/src/app/site/common/components/start/start.component.ts:94]: That might not be necessary. You can simply get the observable from the service and give it to the UI directly using async pipe and translate pipe. Youd also gain a translateable welcome text on the fly

**Review by tsiegleauq — CHANGES_REQUESTED** [review]

File: client/src/app/site/common/components/start/start.component.ts (hunk 1 above)
Comment (GabrielInTheWorld) [reply client/src/app/site/common/components/start/start.component.ts]: It works better without `combineLatest`.

**Review by tsiegleauq — APPROVED** [review]

**Pull request merged**